chain/epoch_manager/src/lib.rs

```diff
@@ -917,7 +917,7 @@ impl EpochManager {
         Ok(EpochId(first_block_info.prev_hash))
     }
 
-    fn get_epoch_info(&mut self, epoch_id: &EpochId) -> Result<&EpochInfo, EpochError> {
+    pub fn get_epoch_info(&mut self, epoch_id: &EpochId) -> Result<&EpochInfo, EpochError> {
```

Comment:
I thought the idea is to not make this public. If we do then we can get rid of a lot of method in epoch manager can directly implement them in runtime adapter.

Reply:
Might be. I couldn't find a reason now to not expose it. We can clean up runtime adapter separately.